### PR TITLE
Add data download script

### PR DESCRIPTION
**Pull request by hombit**

### CODE REVIEW

File: zwad/download_feature_set.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    # but we'd like to practice a bit...
+
+    url = description['links']['self']
+
+    dest = os.path.join(destdir, description['key'])
+    expected_size = description['size']
+
+    # Download resuming is not possible at the moment.
+    # Zenodo doesn't support it. So let's just test the size for equality.
+    # See: https://github.com/zenodo/zenodo/issues/1599
```
Comment (matwey) [inline zwad/download_feature_set.py:40]: These comments are copy-paste from some tutorial, or do you want to say something?

File: zwad/download_feature_set.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```
Comment (matwey) [inline zwad/download_feature_set.py:1]: We use `#!/usr/bin/env python` here in other files.

File: README.md [hunk 3]
```diff
@@ -22,12 +22,11 @@ pip install -e .
 ```
 
 ## Data download
-Light curve feature data for ZTF DR3 fields used in the research are available on SNAD SAI MSU server:
+Light curve feature data for ZTF DR3 fields used in the research are available on [Zenodo](https://zenodo.org/record/4318700).
+You can download it by:
```
Comment (matwey) [inline README.md:26]: Please note, that the script is a convenient way to download the data, but not the single one possible way to do so.

File: zwad/download_feature_set.py (hunk 2 above)
Comment (hombit) [reply zwad/download_feature_set.py:1]: https://github.com/snad-space/zwad/blob/9204d31aedd3878892c46ec0b248da870919491d/zwad/nn.py#L1

File: zwad/download_feature_set.py (hunk 1 above)
Comment (hombit) [reply zwad/download_feature_set.py:40]: Ask @balodja

File: zwad/download_feature_set.py (hunk 2 above)
Comment (matwey) [reply zwad/download_feature_set.py:1]: https://github.com/snad-space/zwad/blob/9204d31aedd3878892c46ec0b248da870919491d/zwad/aad.py#L1

Which one is the right?

**Pull request merged**